pipeline/extract.py

```diff
@@ -13,7 +13,8 @@ def get_top_stories(count: int) -> list:
         top_stories = get(BASE_URL + "topstories.json", timeout=100).json()
         return top_stories[:count]
     except exceptions.RequestException as e:
-         logging.error(f"Error getting api request {e}")
+        logging.exception(f"Error getting api request {e}")
```

Comment:
Double-heck me on this with a pylint run - but I think pylint prefers you don't use f-strings here.

If that's the case, you can use the following syntax:
`logging.exception("Error getting api request %s", e)`

Reply:
I'll make these corrections for you actually!